infra/variables.tf

```diff
@@ -0,0 +1,46 @@
+// Common Variables
+variable region {
+  description = "AWS region to provision infrastrucutre"
+  default     = "us-east-1"
+}
+
+variable "name" {
+  description = "Solution name, e.g. 'app' or 'ecs'"
+}
+
+variable "namespace" {
+  description = "Organization namespace"
+}
+
+variable environment {
+  description = "Name of environment (dev, stable, production, etc)"
+}
+
+variable "tags" {
+  type        = map(string)
+  description = "Tags for infrastructure"
+  default     = {}
+}
+
+// Networking
+variable availability_zones {
+  description = "List of AZ used for provisioning network. Best practice to provide at least 2."
+}
+
+variable "vpc_cidr_block" {
+  description = "CIDR block for VPC"
+}
+
+variable "domain" {
```

Comment:
What's this ultimately used for? The root DNS for the ALB?

Reply:
It's used for looking up the SSL cert and for a few container configs (ex 0x). Also future-proof for when we manage DNS in code.